test/column/renderEditCell.test.tsx

```diff
@@ -16,25 +16,25 @@ describe('Editor', () => {
   it('should open editor on double click', async () => {
     render(<EditorTest />);
     await userEvent.click(getCellsAtRowIndex(0)[0]);
-    expect(screen.queryByLabelText('col1-editor')).not.toBeInTheDocument();
+    expect(screen.queryByRole('spinbutton', { name: 'col1-editor' })).not.toBeInTheDocument();
```

Comment:
What element are we actually querying here? I don't really understand what a spinbutton is

Reply:
`input type=number`